ghedesigner/validate.py

```diff
@@ -5,144 +5,36 @@
 
 from jsonschema import ValidationError, validate
 
-from ghedesigner.enums import BHPipeType, DesignGeomType
-
 # Note: JSON schema does not currently have a good way to handle case-insensitive enums.
-#       Some fields are upper-cased manually here for validation purposes.
+#       I think we should enforce case sensitivity.  The validation script should clearly alert the user.
 #       More details here: https://github.com/json-schema-org/community/discussions/148
 
 
-def validate_schema_instance(schema_file_name: str, instance: dict, error_msg: str) -> int:
+def validate_input_file(input_file_path: Path) -> int:
```

Comment:
The whole validation file is just this one function, since the schema is one file.  There are new items in the schema which allow switching between required inputs based on a choice of one field.